### PR TITLE
Standardize Prow Config 

### PR DESCRIPTION
**Pull request by dustman9000**
This brings the certman-operator prow config up-to-date with expected boilerplate standards.

Depends on https://github.com/openshift/certman-operator/pull/170 to be merged first.

### CODE REVIEW

**Comment by 2uasimojo** [comment]
From a boilerplate perspective,

/lgtm

Will need to be stamped by an approver from OWNERS (once retested after the dep merges).

**Comment by 2uasimojo** [comment]
/retest

dep merged

**Review by jharrington22 — APPROVED** [review]
/lgtm

**Comment by 2uasimojo** [comment]
/retest

[Fix](https://github.com/openshift/certman-operator/pull/172) merged.

**Comment by 2uasimojo** [comment]
TL;DR: This is ready to go.

/assign @yithian 

The latest [lint](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14354/rehearse-14354-pull-ci-openshift-certman-operator-master-lint/1349367674680905728) and [validate](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14354/rehearse-14354-pull-ci-openshift-certman-operator-master-validate/1349367674894815232) rehearsals succeeded, and [pj-rehearse](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14354/pull-ci-openshift-release-master-pj-rehearse/1349367455813734400) is showing success. Not sure why we're still seeing yesterday's results linked here in the PR.
